src/components/ImageCarousel/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactComponent as NextIcon } from 'assets/svg/shop/next-arrow.svg';
+import { ReactComponent as NotFoundImageIcon } from 'assets/svg/shop/not-found.svg';
+import { ReactComponent as PrevIcon } from 'assets/svg/shop/prev-arrow.svg';
+import useCounter from 'utils/hooks/useCounter';
+import makeToast from 'utils/ts/makeToast';
+
+import styles from './ImageCarousel.module.scss';
+
+interface Props {
+  pathname: string;
+  imageUrls: string[] | null | undefined;
+}
```

Comment:
```suggestion
interface Props {
  pathname: 'shop' | 'pin';
  imageUrls: string[] | null | undefined;
}
```
이런 식으로 쓸 수 있을 것 같아요!😀 혹시 pathname으로 받는 값이 shop, pin 말고 더 있나요?
그리고 pathname 대신 pathName이나 path는 어떻게 생각하시나요?